src/components/MatchList/MatchList.style.ts

```diff
@@ -0,0 +1,32 @@
+import styled from '@emotion/styled';
+
+export const Container = styled.div`
+  width: 100%;
+  padding: 70px 0;
+  z-index: -1;
+`;
+
+export const Category = styled.div`
+  text-align: center;
+  width: 100%;
+  height: 100px;
+  padding: 20px 10px 0 7px;
+  overflow: auto;
+  position: fixed;
+  white-space: nowrap;
+  ::-webkit-scrollbar {
+    display: none;
+  }
+  background-color: ${({ theme }) => theme.color.background}; ;
+`;
+
+export const ListContainer = styled.div`
+  margin: 0 20px;
+  padding-top: 110px;
+`;
+
+export const ButtonContainer = styled.div`
+  position: fixed;
+  bottom: 100px;
+  left: 134px;
```

Comment:
```suggestion
  left: 50%;
  transform: translateX(-50%)
```
지금은 기기 너비에 따라서 버튼이 완벽하게 가운데 정렬이 안될 수 있을 것 같습니다. 위와 같이 변경하면 모든 기기에서 동일하게 가운데 정렬 할 수 있을 것 같습니다!

Reply:
리뷰 감사해요 정기님~!!